src/utils/constants.js

```diff
@@ -0,0 +1,138 @@
+export const ALERT_TYPES = [
+    'warning',
+    'error',
+    'success',
+    'information'
+];
+
+export const BADGE_TYPES = [
+    'success',
+    'warning',
+    'error'
+];
+export const LABEL_TYPES = [
+    'success',
+    'warning',
+    'error'
+];
+
+export const STATUS_TYPES = [
+    'success',
+    'warning',
+    'error',
+    'available',
+    'away',
+    'busy',
+    'offline'
+];
+
+export const BUTTON_OPTIONS = [
+    'emphasized',
+    'light',
+    'shell'
+];
+export const BUTTON_TYPES = [
+    'standard',
+    'positive',
+    'negative',
+    'medium'
+];
+
+export const FORM_MESSAGE_TYPES = [
+    'error',
+    'warning',
+    'help'
+];
+
+
+export const ICON_SIZES = [
+    's',
+    'm',
+    'l',
+    'xl'
+];
+
+export const IDENTIFIER_COLORS = [
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9
+];
+
+export const IDENTIFIER_MODIFIERS = [
+    '',
```

Comment:
I don't like that ` ` is a valid value. lets give it a name (default? rectangular? none?) and operate on that.

Reply:
I'd actually say let's just leave it out.  The value `''` is going to evaluate as falsy anyway.  Besides, if we added a different constant and made it the default value, it would generate a CSS classname that doesn't exist.